### DISCUSSION TITLE
Customising Tabby for specific code type

### DISCUSSION THREAD

**Discussion by Gnomesenpai** [TabbyML/tabby#3494] [General]
Hey all,

I run a small community for TF2 MvM, we make custom missions and whatnot for our community to play, we have tried Tabby and context but it misses the ball 99% of the time of what you actually want it to code and i dont think .pop files we work with arent in the code sets that Starcoder is based on. How in the wold can I start with something more fine tuned for our missions? I'm very much a newbie when it comes to AI/LLMs.

tl;dr - I want to input our existing .pop files into a model to help create new missions.
